app/views/reports/new.html.erb

```diff
@@ -2,17 +2,24 @@
 
 <%= simple_form_for @report do |form| %>
   <%= form.association :recipient, label_method: :first_name, value_method: :id %>
-    <% KeyTrait::CATEGORIES.each do |key_trait| %>
-      <h2><%= key_trait.titleize %></h2>
+    <% @skills_data.each do |key_trait, skill_groups| %>
+      <h2><%= key_trait %></h2>
       <%= form.simple_fields_for :key_traits, KeyTrait.new() do |key_trait_form| %>
         <%= key_trait_form.input :category, as: :hidden, input_html: { value: key_trait } %>
-        <% SkillGroup::CATEGORIES[key_trait.parameterize.underscore.to_sym].each do |skill_group| %>
+        <% skill_groups.each do |skill_group, question_skills_data|%>
```

Comment:
Sorry to be picky again. Could we change "question_skills_data" to "skills_and_question"?